classes/controller/AdminController.php

```diff
@@ -416,6 +416,9 @@ public function __construct($forceControllerName = '', $default_theme_name = 'de
         if (strpos($this->controller_name, 'ControllerOverride')) {
             $this->controller_name = substr($this->controller_name, 0, -18);
         }
+        if (strpos($this->controller_name, 'ControllerCore')) {
```

Comment:
Mmm weird, why do you need this? Not that I don't understand the purpose but then it should have caused a few bugs long before right?

Reply:
Reverted